service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasThresholdDetailsValidator.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.apache.commons.lang.StringUtils;
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasThresholdDetails;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasThresholdDetailsValidator implements
+    ConstraintValidator<HasThresholdDetails, CaseData> {
+
+    @Override
+    public void initialize(HasThresholdDetails constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(CaseData caseData, ConstraintValidatorContext constraintValidatorContext) {
+        constraintValidatorContext.buildConstraintViolationWithTemplate(constraintValidatorContext
+            .getDefaultConstraintMessageTemplate()).addPropertyNode("groundsForTheApplication")
+            .addConstraintViolation();
+
+        if (caseData.getOrders() != null && caseData.getOrders().getOrderType() != null
```

Comment:
Maybe you could helper method somewhere to get order by type? Maybe even helper method inside the caseData? 

You are repeating the same code multiple times.